userspace/libsinsp/test/public_sinsp_API/interesting_syscalls.cpp

```diff
@@ -407,15 +374,17 @@ TEST(interesting_syscalls, get_syscalls_names)
 	ppm_sc_set.insert(PPM_SC_READ);
 	orderd_syscall_names_matching_set.insert("read");
 
-	auto orderd_syscall_names_final_set = test_utils::unordered_set_to_ordered(inspector->get_syscalls_names(ppm_sc_set));
+	auto syscall_names_final_set = libsinsp::events::sc_set_to_names(ppm_sc_set);
```

Comment:
Add one more test re `names_to_sc_set` and include one non syscall event, e.g. `container` in names as well to test that we only retrieve the correct `sc_set`.

Reply:
I think we can improve all of these in a subsequent PR ;) we will be working around the new API anyway!